src/test/java/coupon/coupon/domain/DiscountTest.java

```diff
@@ -0,0 +1,96 @@
+package coupon.coupon.domain;
+
+import coupon.coupon.exception.CouponErrorMessage;
+import coupon.coupon.exception.CouponException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class DiscountTest {
+
+    @Test
+    void createDiscount_withValidPriceAndPercent_shouldSucceed() {
+        // Given
+        int validPrice = 5000;
+        int minimumOrderAmount = 50000;
+        int expectedPercent = 10;
+
+        // When
+        Discount discount = new Discount(validPrice, minimumOrderAmount);
+
+        // Then
+        assertEquals(validPrice, discount.getPrice());
+        assertEquals(expectedPercent, discount.getPercent());
```

Comment:
`assertAll()`로 묶어주면 어떤가요? 중간에 검증이 실패하더라도 중지하지 않아요~

```suggestion
        assertAll(
                () -> assertEquals(validPrice, discount.getPrice()),
                () -> assertEquals(expectedPercent, discount.getPercent()) 
        );
```

Reply:
역시 테스트 고수 조조 ..